### DISCUSSION TITLE
There are no other operations when starting clustermgr, and no volume is created. Why is this error reported?

### DISCUSSION THREAD

**Discussion by biao-lvwan** [cubefs/cubefs#3591] [Q&A]
2024/12/07 16:48:25.408731 [ERROR] blobstore/clustermgr/diskmgr/alloc.go:155 [526e79ab29045534/1:ebea2f76b0faa00b] can not find nodeset of diskType: hdd
2024/12/07 16:48:25.408760 [ERROR] blobstore/clustermgr/diskmgr/alloc.go:89 [526e79ab29045534/1:ebea2f76b0faa00b] alloc nodeset failed, err: no enough space to alloc
2024/12/07 16:48:25.408767 [ERROR] blobstore/clustermgr/diskmgr/diskmgr.go:507 [526e79ab29045534/1:ebea2f76b0faa00b] create volume alloc first time failed, err: no enough space to alloc
2024/12/07 16:48:25.408801 [ERROR] blobstore/clustermgr/volumemgr/volumemgr.go:784 [253c5d7d186b99b3:c52cb5bc12ae26fb] create volume failed ==>  --> blobstore/clustermgr/volumemgr/createvolume.go:162 no enough space to alloc ~ alloc chunk for volume[1] unit failed

**Comment by sejust** [comment]
First, start the blobnode and wait for the disks to register; then the clustermgr can create the volume.
